### ISSUE TITLE
StringBuilder missing some Append overloads

### ISSUE THREAD

**Issue by jwosty** [fable-compiler/Fable#2636]
### Description

```StringBuilder.Append``` is missing replacements for the float and decimal overloads. There may be more; those two are all I checked.

```fsharp
stringBuilder.Append (string someFloatValue)
```

### Related information

* Fable version: 3.1.4
* Operating system: Windows 10.0.19043

**Comment by alfonsogarciacaro** [comment]
Yes, atm not all overloads of `Append` are supported. If you want to add more, it should be possible to just add the overload to the replacement here: https://github.com/fable-compiler/Fable/blob/1dfdd94c54375fb6c82bfff5ed93b0a031bfc0cc/src/fable-library/System.Text.fs#L3-L16

And tests here: https://github.com/fable-compiler/Fable/blob/1dfdd94c54375fb6c82bfff5ed93b0a031bfc0cc/tests/Main/StringTests.fs#L80-L88

**Comment by jwosty** [comment]
Yep, seems easy enough. Expect a PR soon
